chemprop/uncertainty/uncertainty_calibrator.py

```diff
@@ -171,14 +178,18 @@ def nll(
         preds: List[List[float]],
         unc: List[List[float]],
         targets: List[List[float]],
+        mask: List[List[bool]],
     ):
         unc_var = np.square(unc)
         preds = np.array(preds)
-        targets = np.array(targets)
-        return (
+        targets = np.array(targets, dtype=float)
+        mask = np.array(mask, dtype=bool)
+        nll = (
             np.log(2 * np.pi * unc_var) / 2
             + (preds - targets) ** 2 / (2 * unc_var)
         )
+        nll = np.where(mask, nll, 0.)
```

Comment:
For consistency, would it be better to replace all the np.where() with removing the masked values. Besides, these nll values will be averaged in each task by evaluate() in uncertainty_evaluator.py. I think we should remove these nll values instead of treating them as zeros and further be averaged. My suggestion would be to replace all the existing np.where() with removing these values in both calibrator.py and uncertainty_evaluator.py.

Reply:
You are totally right. I will make this change.